<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Introduction</class>
 <widget class="QWidget" name="Introduction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>482</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>501</width>
     <height>288</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>288</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>288</height>
    </size>
   </property>
   <widget class="QLabel" name="labelWallpaper">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>165</width>
      <height>289</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>285</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>165</width>
      <height>289</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditcontent">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>330</width>
      <height>223</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>223</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>223</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleIntroduction">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>292</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>292</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
